bciers/libs/components/src/form/MultiStepBase.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
+
+import { useState } from "react";
+import { useRouter } from "next/navigation";
+import { Button } from "@mui/material";
+import { RJSFSchema, UiSchema } from "@rjsf/utils";
+import { Alert } from "@mui/material";
+import FormBase from "./FormBase";
+import MultiStepHeader from "./components/MultiStepHeader";
+import MultiStepButtons from "./components/MultiStepButtons";
+import { IChangeEvent } from "@rjsf/core";
+
+interface MultiStepBaseProps {
+  allowBackNavigation?: boolean;
```

Comment:
We had discussed this at one point and I thought the expectation was that the user would be able to check and edit previous pages during submission. We should clarify that - it would be much easier if they couldn't (no PUT routes). That said, I think for future reuse we can keep this in.

Reply:
Most of the registration endpoints are going to be PUTs because we're updating the operation model